modules/liveIntentIdSystem.js

```diff
@@ -107,6 +107,12 @@ function initializeLiveConnect(configParams) {
   }
 
   const liveConnectConfig = parseLiveIntentCollectorConfig(configParams.liCollectConfig);
+
+  if (!liveConnectConfig.appId && configParams.distributorId) {
+    liveConnectConfig.distributorId = configParams.distributorId;
+    identityResolutionConfig.source = configParams.distributorId;
```

Comment:
`identityResolutionConfig.source` is set only if no `appId` was provided. Is that intentional?

Reply:
Yes, it is intentional.